fenix/benchmark/src/main/java/org/mozilla/fenix/benchmark/utils/MacroBenchmarkRule.kt

```diff
@@ -0,0 +1,33 @@
+package org.mozilla.fenix.benchmark.utils
+
+import androidx.annotation.IntRange
+import androidx.benchmark.macro.CompilationMode
+import androidx.benchmark.macro.MacrobenchmarkScope
+import androidx.benchmark.macro.Metric
+import androidx.benchmark.macro.StartupMode
+import androidx.benchmark.macro.junit4.MacrobenchmarkRule
+
+/**
+ * Extension function that calls [MacrobenchmarkRule.measureRepeated] with
+ * defaults parameters set for [packageName] and [iterations].
+ */
+fun MacrobenchmarkRule.measureRepeatedX(
+    packageName: String = TARGET_PACKAGE,
+    metrics: List<Metric>,
+    compilationMode: CompilationMode = CompilationMode.DEFAULT,
+    startupMode: StartupMode? = null,
+    @IntRange(from = 1)
+    iterations: Int = DEFAULT_ITERATIONS,
+    setupBlock: MacrobenchmarkScope.() -> Unit = {},
+    measureBlock: MacrobenchmarkScope.() -> Unit,
```

Comment:
nit: I think it could make sense to split this into two functions, `measureRepeatedDefault` and `measureRepeatedX`(with no default iterations) just to increase clarity around iterations. That, or I'd consider just dropping the `X` from the name.

Reply:
Yeah `measureRepeatedDefault` does read better, will update that 👍  The idea was to add one extension method with default params, there is already a `measureRepeated` function on `MacroBenchmarkScope` without the app specific defaults that this function calls